modules/java-modules/http/src/main/java/org/syslog_ng/http/HTTPDestinationOptions.java

```diff
@@ -32,6 +32,10 @@
 public class HTTPDestinationOptions {
 
   public static String URL = "url";
+  public static String METHOD  = "method";
+  public static String METHOD_DEFAULT  = "PUT";
+
+  public static HashSet<String> HTTP_METHODS = new HashSet<String>(Arrays.asList("POST", "PUT"));
```

Comment:
What about the other methods (HEAD. DELETE, ...)? Maybe we should really let the user use templates and if it means to shoot herself in the foot then let her do it.